docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,149 @@
+
+**Example**
+
+The following *artifact.xml* file is produced after running the example command shown above. Note that [`Castle.Core`](https://www.nuget.org/packages/Castle.Core/) is a depdenency of `Moq`, so it's included automatically and appears in the *artifacts.xml* manifest file.
+
+```xml
+<StoreArtifacts>
+  <Package Id="newtonsoft.json" Version="10.0.3" />
+  <Package Id="castle.core" Version="4.1.0" />
+  <Package Id="moq" Version="4.7.63" />
+</StoreArtifacts>
+```
+
+## Publishing an app against a target manifest
+
+If you have a target manifest file on disk, you specify the path to the file when publishing your app with the [`dotnet publish`](../tools/dotnet-publish.md) command:
+
+```console
+dotnet publish --manifest <PATH_TO_MANIFEST_FILE>
+```
+
+**Example**
+
+```console
+dotnet publish --manifest manifest.xml
+```
+
+You're required to deploy the resulting published app to an environment that has the packages described in the target manifest. Failing to do so results in the app failing to start.
+
+Specify multiple target manifests when publishing an app by repeating the option and path (for example, `--manifest manifest1.xml --manifest manifest2.xml`). When you do so, the app is trimmed for the union of packages specified in the target manifest files provided to the command.
+
+## Specifying target manifests in the project file
+
+Instead of specifying target manifests with the [`dotnet publish`](../tools/dotnet-publish.md) command, you can specify them in the project file as a semicolon-separated list of paths under a **\<TargetManifestFiles>** tag.
+
+```xml
+<PropertyGroup>
+  <TargetManifestFiles>manifest1.xml;manifest2.xml</TargetManifestFiles>
+</PropertyGroup>
+```
+
+Specify the target manifests in the project file only when the target environment for the app is well-known, such as for .NET Core projects. This isn't the case for open-source projects. The users of an open-source project likely deploy to different production environments, which probably have different sets of packages pre-installed. Maintainers of the project can't make assumptions about the target manifest in such open-source environments, so developers should instead rely on the `--manifest` option of [`dotnet publish`](../tools/dotnet-publish.md).
+
+## ASP.NET Core implicit store
+
+The runtime package store feature is used implicitly by ASP.NET Core apps. The targets in [`Microsoft.NET.Sdk.Web`](https://github.com/aspnet/websdk) include manifests referencing the implicit package store that's available when the .NET Core SDK is installed on the target system. Running `dotnet publish` for an app results in a published app that contains only the app and its assets and not the packages required ASP.NET Core targets, which are present on the target system.
+
```

Comment:
See 3:10 into [this video](https://www.dropbox.com/s/a8yscvf4uep5qx8/Metapackage%20%2C%20%20runtime%20store%20and%20publ.%20.%20.%20-%20Thursday%2C%20July%2027%2C%202017%202.33.50%20PM.mp4?dl=0)
The runtime store can be added by installers other than the .NET Core SDK. For example: 
* [apt-get](https://help.ubuntu.com/community/AptGet/Howto)
* [redhat yum](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Deployment_Guide/sec-Installing.html) installer
* windows hosting bundle
* downloading zips
* downloadable if you want to install it manually